src/main/webapp/js/datahandlers/listdatahandler.js

```diff
@@ -68,6 +68,7 @@ class ListDataHandler {
           await fetch(this.getPath_(batchIndex, itemsPerBatch, lastIndex));
       let listJson = await responseList.json();
       if (type == COLLEGES) {
+        console.log(listJson);
```

Comment:
Might want to remove this for devel if it was just used for debugging.

Reply:
Yep just deleted it.